Tensile/KernelWriterAssembly.py

```diff
@@ -6647,43 +6663,101 @@ def mfmaIter(self, kernel, m, innerUnroll, vregSetIdx, lastKinloop=False, tail=F
             ccVgprs = [None]*3 # three terms that can be negated: [real1, imag0, imag1]
             ccInsts = [None]*3
             accImOffset = self.AccVgprImagNumOffset(kernel)
+            # for firstIter, need to use accStartSrc for img instead of adding accImOffset
+            accStartSrcImg1 = accStartSrc1 if firstIter else accStartSrc1+accImOffset
```

Comment:
What are the considerations for needing this special `firstIter == true` case?

Reply:
This is to optimize accreg initialization.
Instead of initializing all accregs, only 8 accregs are initialized.
At the first iteration, the initialized 8 accregs are used as source accreg.